raiden/api/python.py

```diff
@@ -82,10 +88,10 @@ def get_channel(self, registry_address, token_address, partner_address):
 
     def token_network_register(
             self,
-            registry_address,
-            token_address,
-            poll_timeout=DEFAULT_POLL_TIMEOUT,
-            retry_timeout=DEFAULT_RETRY_TIMEOUT,
+            registry_address: typing.PaymentNetworkID,
+            token_address: typing.TokenAddress,
+            poll_timeout: int = DEFAULT_POLL_TIMEOUT,
+            retry_timeout: float = DEFAULT_RETRY_TIMEOUT,
```

Comment:
We have a type for timeouts named [NetworkTimeout](https://github.com/raiden-network/raiden/blob/bf8da6c0a362416682e5b42d36800f4e0e7c2d46/raiden/utils/typing.py#L54).

Also, [`poll_timeout` is a float](http://www.gevent.org/api/gevent.timeout.html) not a int.